site/pages/type-of-consequence.page.tsx

```diff
@@ -0,0 +1,112 @@
+import { NextPageContext } from "next";
+import { destroyCookie, parseCookies } from "nookies";
+import { ReactElement, useState } from "react";
+import { inspect } from "util";
+import Radios from "../components/form/Radios";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+import { ConsequenceType, TransportMode } from "../constants/enum";
+import { COOKIES_ADD_CONSEQUENCE_ERRORS, COOKIES_ADD_CONSEQUENCE_INFO } from "../constants/index";
+import { AddConsequenceProps, AddConsequenceWithErrors, DisplayValuePair, ErrorInfo } from "../interfaces/index";
+import logger from "../utils/logger";
+
+const title = "Create Consequences";
+const description = "Create Consequences page for the Create Transport Disruptions Service";
+
+const modeOfTransportRadio: DisplayValuePair[] = [];
+
+Object.values(TransportMode).forEach((enumValue) => {
+    modeOfTransportRadio.push({
+        value: enumValue,
+        display: enumValue,
+    });
+});
+
+const consequenceType: DisplayValuePair[] = [];
+
+Object.values(ConsequenceType).forEach((enumValue) => {
```

Comment:
Same here - should we not be using Object.entries and then using the lowercase as the value and the sentence case as the display?

Reply:
done